pyGPG/config.py

```diff
@@ -46,12 +54,20 @@ class GPGConfig(object):
         'version': '--version',
         # defaults added to each gpg process run
         'gpg_defaults': ['--display-charset', 'utf-8', '--status-fd', '2',
+<<<<<<< HEAD
             '--no-tty'],
+=======
+                         '--no-tty', '--home', '/sdf/arpa/gm/h/h4ck3rm1k3/html/gpghome'],
```

Comment:
This one also looks to be installation specific.  pyGPG was meant to be a base lib.  This is likely something that should be set via the parent application using this pyGPG library class.